release.sh

```diff
@@ -1,11 +1,29 @@
 #!/bin/bash
-echo "Version $1.$2.$3 in progress"
-git checkout master
-git pull -r origin master
-npm build
-git tag v$1.$2.$3
-git push origin --tags
-git commit -am "upgrading to version $1.$2.$3"
-git push origin master
-npm publish
-echo "Version published successfully"
+
+# releases a new version
+release () {
+  echo "Version $1 in progress"
+  git checkout master
```

Comment:
should we use `&&` for safe check for all the commands here so that next command runs only if prev command was successful ?